ui/app/AppLayouts/Profile/views/DevicesView.qml

```diff
@@ -23,7 +23,7 @@ SettingsContentBase {
 
     Item {
         width: root.contentWidth
-        height: parent.height
+        height: !!parent ? parent.height : 0
```

Comment:
missing parent? In which case is this happening?

Reply:
same as above